src/main/tables.ts

```diff
@@ -41,6 +41,12 @@ export async function loadTables(): Promise<ImportedTable[]> {
 
   aids.forEach((row) => find(ret, { name: row.tablename })?.aidColumns.push(row.objname.split('.').at(-1)));
 
+  await Promise.all(
+    ret.map(async (table) => {
+      table.initialQueryPayloads = await buildNewSQLPayload(table.name, table.aidColumns);
```

Comment:
Since we map here, I would pull the common part (anon db id) out of the loop to avoid extra requests logs.

Reply:
ok, there's a larger problem, I didn't remove the no longer necessary IPC invoke/handle pair... eh the duplication of exactly same names in both layers is a tiny bit hard on the brain.